index.html

```diff
@@ -132,14 +133,15 @@ <h2>About Myself</h2>
                 </ul>
                 <button class="about-btn btn">Get my Resume</button>
             </div>
+        <div class="child-about">
                 <div class="about-languages">
                         <img id="about-languages1" class="iconss" src="img2/angle-down.png" alt="">
                         <P>Languages</P>
                 </div>
                 <div class="container">
                     <div class="lang-icon">
                         <div class="lang">
-                            <img id="im" src="img2/Ellipse 1.png" alt="javascript">
+                            <img id="js-icon" src="img2/Ellipse 1.png" alt="javascript">
                             <p>Javascript</p>
                         </div>
                         <div class="lang">
```

Comment:
Maybe you can use a `<ul>` for the technologies. It helps you to order the informtation more properly